articles/references/templates_decorators/decorator.md

```diff
@@ -0,0 +1,119 @@
+---
```

Comment:
Maybe we all go through this file together on Monday evening.

If I recall correctly, @garethbaars is now of the opinion that for the Startup file, events are a better option than decorators. In which case we shouldn't be using that as an example use case.

Reply:
I know this is a side discussion but I am of the opinion that events are problematic in some scenarios. With that said, even if events are "superior", I would still keep this how-to as is, explaining how the decorators inject code into the startup file since it covers pretty much all the functionality of decorators in a demonstrable way.

Reply:
Let's raise with G on Thursday evening.